packages/api/src/eventListener.ts

```diff
@@ -780,6 +794,12 @@ export default class EventListener extends IndexPriceInterface {
 
 		// update data in sdkInterface's exchangeInfo
 		const fundingRate = this.fundingRate.get(perpetualId) || 0;
+		// Do not allow 0 funding rate to be sent out
+		if (fundingRate === 0) {
+			console.log(`[onUpdateMarkPrice] funding rate is 0 for ${perpetualId}`);
+			return;
```

Comment:
this should be an error. Because if the code is correct we never run into this situation

Reply:
adjusted